src/components/SearchNavBar.tsx

```diff
@@ -0,0 +1,197 @@
+import React from 'react';
```

Comment:
It looks like there's a lot of unnecessary styling + extra components here! Try to keep the build as close to [Ashley's design](https://www.figma.com/file/bMM7HMsbgcXWLIdGV30B6b/Mee-Panyar?node-id=406%3A1) as possible.

Also update styling to the styled-components (you can look at my comments/changes regarding the BottomNav as an example)